packages/split-ssh/src/split-ssh-formula/state/policy/init.sls

```diff
@@ -5,4 +5,5 @@
     - mode: '0755'
     - makedirs: True
     - source: salt://split-ssh/policy/files/qubes.SSHAgent
+    - template: jinja
```

Comment:
This is the default for `file.managed`, but I think being explicit makes it clearer.